api/src/utils/preprocess_incidents.js

```diff
@@ -0,0 +1,93 @@
+/** Preprocess CSV data for 
+** 1) the count of incidents for each incident type
+** 2) the count of incidents per state
+** 3) the count of incidents per year
+**/
+
+var csv = require('fast-csv');
+const dateParser = require('./date_parser');
+
+var stateCounts = {}; // key-value pairs of state->incident count
+var yearCounts = {}; // key-value pairs of year(19,20...)->incident count
+var incidentCounts = {}; // key-value pairs of incident type(human trafficking, massage parlor...)-incident count
+
+csv.parseFile("./api/src/utils/prostitution_arrests_incidents.csv", {headers: true})
```

Comment:
So we don't need to reference any actual file since we will just interact with the data itself from the upload endpoint.  Let's turn these on data callbacks into functions and export them